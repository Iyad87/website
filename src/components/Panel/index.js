import styled from 'styled-components';
import { spacing } from '../../styles/spacing';

export const panelPadding = spacing(3);

const Panel = styled.div`
  background-color: white;
  border-radius: 4px;
  box-shadow: 0 3px 20px rgba(161, 160, 197, 0.2);
  padding: ${panelPadding};
`;

export default Panel;
