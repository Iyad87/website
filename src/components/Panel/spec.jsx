import React from 'react';
import { shallow } from 'enzyme';
import Panel from './index';

test('components/Panel snapshot', () => {
  const wrapper = shallow(<Panel />);

  expect(wrapper).toMatchSnapshot();
});
