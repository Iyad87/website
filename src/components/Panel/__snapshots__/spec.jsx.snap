// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Panel snapshot 1`] = `
<div
  className="sc-bdVaJa eTNNel"
/>
`;
